src/AppBundle/Form/SitesFilterType.php

```diff
@@ -24,6 +25,17 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'attr' => ['placeholder' => 'Name'],
                 'required' => false,
             ])
+            ->add('client', EntityType::class, [
+                'class' => Client::class,
+                'choice_label' => function ($client){
```

Comment:
Use a typehint i.e. `function (Client $client) {`

As much as possible typehint functions, and avoid `@var`